app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/home/HomeCategoryScreen.kt

```diff
@@ -116,10 +119,25 @@ fun HomeCategoryScreen(
                     CategoryCard(category) { onNavigateToCategory(category.title) }
                 }
             }
+            CheckAccessToken()
         }
     }
 }
 
+@Composable
```

Comment:
Maybe just a function level comment here to remind us what this is doing in a few years time :D